Libraries/CustomComponents/NavigationExperimental/NavigationCardStack.js

```diff
@@ -120,7 +117,6 @@ class NavigationCardStack extends React.Component {
         navigationState={this.props.navigationState}
         renderOverlay={this.props.renderOverlay}
         renderScene={this._renderScene}
-        applyAnimation={this._applyAnimation}
```

Comment:
Can we add this.props.applyAnimation in a followup, to allow people to customize it?